rollup-bridge-contracts/package.json

```diff
@@ -67,6 +66,7 @@
     "solc": "^0.8.28",
     "solhint": "^5.0.5",
     "solidity-coverage": "^0.8.5",
+    "solmate": "^6.8.0",
```

Comment:
seems it leads CI fail on downloading module, why we need to introduce `solmate` for bridge contracts?

Reply:
@akokoshn solmate contains optimised safe token transfer library, this is used across all protocols and bridges
this is part of public npm library, not sure why its giving error

Reply:
because `npm` was deprecated